### PR TITLE
Create full website

### PR DESCRIPTION
**Pull request by EvansSnave**
-implemented react-redux
-Implemented redux toolkit
-Missions and rockets retrieved from API
-Style similar to the given design.
-Reserve and cancel reservation rocket
-Join and leave missions

### CODE REVIEW

**Review by ademibowale — COMMENTED** [review]
Hi @EvansSnave ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that has no commit or contribution from your coding partner, This milestone is a [pair-programming] kindly contact your partner to work as a team on this milestone and resubmit

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by ademibowale — COMMENTED** [review]
Hi @EvansSnave ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project invalid project with no contribution as this is a pair program project, attach your screenshot that indicates you are alone or evidence from the student success that shows that you don't have a coding partner.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by EvansSnave** [comment]
@ademibowale I have been working ALONE
image

File: src/pages/myProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Table from 'react-bootstrap/Table';
+
+const MyProfile = () => {
+  const rockets = useSelector((state) => state.rockets);
+  const missions = useSelector((state) => state.missions.missions);
+  const reservedRockets = rockets.filter((rocket) => rocket.reserved);
+  const joinedMissions = missions.filter((mission) => mission.reserved);
```
Comment (ademibowale) [inline src/pages/myProfile.jsx:9]: - [ ] Hello you have done a great job on this milestone ,i clone your project severally with different ideal but your page not display or render anything on screen ,after i check your code base i can see that you didnt import the myProfile in your myProfile.jsx .so this make the page nor render kindly fix it

File: src/components/navigation.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { NavLink } from 'react-router-dom';
+import logo from '../assets/logo.png';
+import '../styles/navigation.css';
+
+function NavBar() {
+  return (
+    <nav>
+      <div className="parentcontainer300">
+        <div className="container300">
+          <img className="logo300" src={logo} alt="Space Traveler" />
```
Comment (ademibowale) [inline src/components/navigation.jsx:10]: - [ ] From the missionList.jsx you have not import the file fromthe components kindly fix it to make your page render and visible it display not on the screen when i clone it severally ,please kindly fix it.

File: src/components/MissionList.jsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+import Button from 'react-bootstrap/Button';
+import Badge from 'react-bootstrap/Badge';
+import { useDispatch } from 'react-redux';
+import { reserveMission, leaveMission } from '../redux/missions/missionsSlice';
+import '../styles/MissionList.css';
+
+const MissionList = ({ missions }) => {
+  const dispatch = useDispatch();
+
+  const handleJoinMission = (missionId) => {
```
Comment (ademibowale) [inline src/components/MissionList.jsx:13]: - [ ] From the missionList.jsx youA have not import the file from the components kindly fix it to make your page render and make it visible, not display on the screen when i clone it severally ,please kindly fix it.

**Review by ademibowale — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @EvansSnave , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🥇

- Descriptive Pull Request title and summary. ✔️
- Good use of Github flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions


- [ ] This is a pair program but the project has no contribution fro any coding partner becouse the student work alone and attached evidence letter from the student success 
- [ ] 
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by EvansSnave** [comment]
You took forever to make the review. Thank you for making me repeat the
module.

**Comment by ademibowale** [comment]
No not what you think ,when review any project we dont rush especially a
capstone project like this yours,that's why it's good to submit a project
at a good time you submitted your project at the dead time .so it wasn't my
fault and if I approve it like that mv might detect that which might create
a problem so I have to follow the requirements and make sure the project
works as expected.
I am so sorry assuming there is still time we can both fix any technical
issues and resubmit then get it approve.

**Comment by ademibowale** [comment]
Whenever you work on a project and you are stucked never stay too long on
it without contact people on slack ,so you can fix it together, your
project you are giving 6 days to complete from monday to Saturday yet you
submitted on Saturday last minit .too late

**Comment by ademibowale** [comment]
Hello Kevin  ,
I am just thinking about you because there was a time I was like you at
that stage i understand how you will be feeling to repeated a module, i
have a suggestion for you please .kindly write Marco Lizza the student
success explain how difficult it's for you only to complete the project at
a given period without pearner, how you encounter difficulties during the
project which you resolve by yourself while you contacted  different
developers on slack but many are too busy to help ,request for additional
week or days if possible just a day to resubmit your project that you have
done and completed the project with all changes requested have been
resolved, ask them to give you just Monday to get your project approve that
you cant just afford to repeat the module, repeating a module can disable a
mora making you weak in learning because you so much devoted in learning
and you dont want anything to discourage you, please write the student
success and make sure you tag maco Lizza with the message  ,you send the
mail via the last email you recieve from Marco lizza last by hit the reply
to write him if he can consider you .

Thank you .

**Review by nicupop729 — COMMENTED** [review]
Hello team 👋,

_This is a manual review._

Please note that during pair programming activities, we cannot utilize previously approved old repositories.
Let's make the most of our group project and collaborate to develop this fantastic project.
Please establish a completely new repository and work on it as a team. Each of you should create the rockets, missions, and dragons sections accordingly.
Ensure that the tasks are evenly divided among you using the Kanban board.

Best of luck with your new project! :four_leaf_clover:

Keep rocking 🚀!

**Pull request closed without merging**